package.json

```diff
@@ -14,24 +15,35 @@
     "bitcoinjs"
   ],
   "scripts": {
-    "coverage-report": "nyc report --reporter=lcov",
-    "coverage-html": "nyc report --reporter=html",
-    "coverage": "nyc --check-coverage --branches 90 --functions 90 mocha",
-    "integration": "mocha --timeout 50000 test/integration/",
-    "standard": "standard",
-    "test": "npm run standard && npm run coverage",
-    "unit": "mocha"
+    "build": "tsc -p ./tsconfig.json",
+    "coverage-report": "npm run build && npm run nobuild:coverage-report",
+    "coverage-html": "npm run build && npm run nobuild:coverage-html",
+    "coverage": "npm run build && npm run nobuild:coverage",
+    "format": "npm run prettier -- --write",
+    "format:ci": "npm run prettier -- --check",
+    "integration": "npm run build && npm run nobuild:integration",
+    "nobuild:coverage-report": "nyc report --reporter=lcov",
+    "nobuild:coverage-html": "nyc report --reporter=html",
+    "nobuild:coverage": "nyc --check-coverage --branches 90 --functions 90 --lines 90 mocha",
+    "nobuild:integration": "mocha --timeout 50000 test/integration/",
+    "nobuild:unit": "mocha",
+    "prepare": "npm run build",
+    "prettier": "prettier 'ts_src/**/*.ts' --ignore-path ./.prettierignore",
+    "test": "npm run build && npm run format:ci && npm run nobuild:coverage",
+    "unit": "npm run build && npm run nobuild:unit"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/bitcoinjs/bitcoinjs-lib.git"
   },
   "files": [
-    "src"
+    "src",
+    "types"
   ],
   "dependencies": {
+    "@types/node": "10.12.18",
     "bech32": "^1.1.2",
-    "bip32": "^1.0.0",
+    "bip32": "git+https://github.com/junderw/bip32.git#typeScript",
```

Comment:
Why don't you push your work to original repository and create a pr so anyone can collaborate on it?

Reply:
https://github.com/bitcoinjs/bip32/pull/10

Reply:
Sorry, I have missed it.